portfolio/src/main/webapp/style.css

```
@@ -4,6 +4,134 @@
   width: 650px;
 }
 
-#greeting-container {
+#secondBack {
+  color: rgb(0, 0, 0); 
+  background-color:rgb(251, 255, 6);
+}
+
+#thirdBack{
+  color: rgb(0, 0, 0); 
+  background-color:rgb(255, 255, 255);
+  width: 800px;
+}
+
+.intro-image{
+  background: url(eva-images/queens.jpg) no-repeat center; 
+  background-size: cover;
+  height: 700px;
+  position: relative;
+  /* Add the blur effect */
+  filter: blur(2px);
+  -webkit-filter: blur(2px);
+}
+
+.intro-text {
+  /* Fallback color */
+  background-color: rgba(0,0,0, 0.3); 
+  border: 3px solid #f1f1f1;
+  text-align: center;
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  width: 80%; 
+  transform: translate(-50%, -50%);
+  padding: 10px;
+  color:rgb(255, 255, 255);
+  font-family: "Lucida Console", Courier, monospace;
+}
+
+.quote-image{
+  background: url(eva-images/space.jpg) no-repeat center; 
+  background-size: cover;
+  height: 500px;
+  position: relative;
+}
+
+.quote-text {
+  /* Fallback color */
+  background-color: rgba(0,0,0, 0.3); 
+  text-align: center;
+  position: relative;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+  padding: 10px;
+  color:rgb(255, 255, 255);
+  font-family: "Lucida Console", Courier, monospace;
+}
+
+/* middle part of main page */
+* {
+  box-sizing: border-box;
+}
+
+.column {
+  float: left;
+  width: 50%;
+  padding: 15px;
+}
+/* Clear floats after the columns */
+.row:after {
+  content: "";
+  display: table;
+  clear: both;
+}
+
+#quote-container {
   margin-top: 20px;
 }
+.button {
+    background-color: rgb(0, 0, 0);
+    border: none;
+    color:rgb(255, 241, 40);
+    padding: 10px 25px;
+    text-align: center;
+    text-decoration: none;
+    display: inline-block;
+    font-size: 16px;
+    margin: 4px 2px;
+    cursor: pointer;
+}
+/* button used for github link */
```

```suggestion
/* button used for github link */
```
Could be removed after renaming.